x/callback/keeper/callback.go

```
@@ -27,18 +27,26 @@ func (k Keeper) GetCallbacksByHeight(ctx sdk.Context, height int64) (callbacks [
 }
 
 // ExistsCallback returns true if the callback exists for height with same contract address and same job id
-func (k Keeper) ExistsCallback(ctx sdk.Context, height int64, contractAddress sdk.AccAddress, jobID uint64) (bool, error) {
+func (k Keeper) ExistsCallback(ctx sdk.Context, height int64, contractAddr string, jobID uint64) (bool, error) {
+	contractAddress := sdk.MustAccAddressFromBech32(contractAddr)
```

let's use the `sdk.AccAddressFromBech32`  since we can return an error here.